docs/commands/InModuleScope.mdx

```diff
@@ -73,6 +73,56 @@ the PowerShell session, because the module only exported
 Using InModuleScope allowed this call to
 "PrivateFunction" to work successfully.
 
+### EXAMPLE 2
+
+```powershell
+# The script module:
+
+function PublicFunction
+
+    # Does something
+
+}
+
+function PrivateFunction ($MyParam)
+
+    return $MyParam
+}
```

Comment:
Corrupted. Source is
```
    ```powershell
    # The script module:
    function PublicFunction
    {
        # Does something
    }

    function PrivateFunction
    {
        return $true
    }
```

Reply:
Bug: `#` is detected as a markdown header and separated

Reply:
I think this one is now resolved. Can you confirm?